libs/design-system/panda.config.ts

```diff
@@ -0,0 +1,35 @@
+import { defineConfig } from '@pandacss/dev';
+import { NovuPandaPreset } from './src/panda/NovuPandaPreset';
+
+export default defineConfig({
+  // Whether to use css reset
+  preflight: true,
+
+  // Where to look for your css declarations
+  include: ['./src/**/*.{js,jsx,ts,tsx}', './.storybook/**/*.{js,jsx,ts,tsx}'],
+
+  // Files to exclude
+  exclude: [],
+
+  presets: [NovuPandaPreset],
+
+  /**
+   * Prefixes generated classes with the specified string (e.g. `nv-text_blue`)
+   * https://panda-css.com/docs/references/config#prefix
+   */
+  prefix: 'nv',
```

Comment:
Nice one, a good one for us to customize so that we don't conflict with Customer styles in places like the Content Editor.